cli/app.go

```diff
@@ -53,3 +64,54 @@ func (a *AMP) Status() {
 		fmt.Println("Status")
 	}
 }
+
+// Logs fetches the logs
+func (a *AMP) Logs(cmd *cobra.Command) error {
```

Comment:
This isn't how I intented `app.go` go get used. It is the job of `cmd/amp/main.go`'s job to parse the command line then call `a.Logs(args...)` not `a.Logs(*cobra.command). I didn't intend for the app object to actually have to know about cobra at all. This make the app object more testable. Also, I figure if the implementation of a command takes more than a few lines, there probably should be a separate file for it (ex: `logcommand.go`).